homework01/allan/vector.h

```diff
@@ -0,0 +1,43 @@
+class Vector
+{
+public:
+
+	Vector()
+	:Vector(0,0)
+	{
+	}
+
+	Vector(int x, int y)
+	:m_x(x),
+	 m_y(y)
+	{
+	}
+
+	bool operator==(const Vector& rhs) const
+	{
+		return m_x == rhs.m_x &&
+			   m_y == rhs.m_y;
```

Comment:
Why not on one line? It shouldn't be more than 120 characters? In general one liners are accepted in header file, the rest goes to the cc.

Reply:
Maybe it's a Java convention but I like to see each test on its own line and the logical operator at the end. I find it more obvious